projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/credentials/JobCredentialsService.java

```diff
@@ -22,6 +22,8 @@
 import java.util.Collections;
 import java.util.Optional;
 
+import static graphql.Assert.assertTrue;
```

Comment:
Not sure if ppl want me to avoid using dependencies like this?

Reply:
Yes. Definitely. I am not sure how successful But I am trying to keep each "service" - e.g Execution Service , Data Jobs Service, Deployment Servie decoupled from each other and independent. 

That also means dependencies dedicated for "Data Jobs Service" (like graphql) should not be used outside. Now without proper compile time checks, this is not easy to do so we need to rely on reviews for that. 

But even as a general rule. The less things any module (be it class, package, namespace) depends , the better.

Reply:
Yeah I agree.